vector/src/main/java/im/vector/app/features/roommemberprofile/RoomMemberProfileFragment.kt

```diff
@@ -289,12 +289,21 @@ class RoomMemberProfileFragment @Inject constructor(
     }
 
     private fun handleShareRoomMemberProfile(permalink: String) {
-        startSharePlainTextIntent(
-                fragment = this,
-                activityResultLauncher = null,
-                chooserTitle = null,
-                text = permalink
-        )
+        val inflater = layoutInflater
+        val view = inflater.inflate(R.layout.item_share_qr_code, null)
+        val qrCode = view.findViewById<im.vector.app.core.ui.views.QrCodeImageView>(R.id.itemShareQrCodeImage)
+        qrCode.setData(permalink)
+        AlertDialog.Builder(requireContext())
+            .setView(view)
+            .setNeutralButton(R.string.ok) { _, _ -> }
```

Comment:
You can simply write `.setNeutralButton(R.string.ok, null)`, this is equivalent, and (I think) a bit less ugly :)

Reply:
It is less ugly indeed. Done :)